src/sdam/topology.ts

```diff
@@ -175,6 +176,7 @@ export interface SelectServerOptions {
   serverSelectionTimeoutMS?: number;
   session?: ClientSession;
   operationName: string;
+  previousServer?: ServerDescription;
```

Comment:
(non-blocking) 

the name `previousServer` is odd when we're performing server selection for the first time, since there's no previous server that has been selected.  what would you think of the more general name, `serversToIgnore: ServerDescription[]`?  We would set it to `[]` for non-retries.

Reply:
I chose `previousServer` since that matches the naming in the spec itself, and figured for clarity it would be fine to keep that name:  https://github.com/mongodb/specifications/blob/master/source/retryable-reads/retryable-reads.rst#pseudocode